config-overrides.js

```diff
@@ -1,18 +1,37 @@
+const CspHtmlWebpackPlugin = require('csp-html-webpack-plugin')
 const _ = require('lodash/fp')
 const rewireReactHotLoader = require('react-app-rewire-hot-loader')
-const { addBabelPlugin } = require('customize-cra')
+const { addBabelPlugin, addWebpackPlugin } = require('customize-cra')
 
 
 module.exports = {
   webpack(config, env) {
-    const newConfig = addBabelPlugin([
-      'prismjs', {
-        languages: ['bash', 'python'],
-        plugins: ['line-numbers'],
-        theme: 'default',
-        css: true
-      }
-    ])(config)
+    const newConfig = _.flow(
+      addBabelPlugin([
+        'prismjs', {
+          languages: ['bash', 'python'],
+          plugins: ['line-numbers'],
+          theme: 'default',
+          css: true
+        }
+      ]),
+      addWebpackPlugin(new CspHtmlWebpackPlugin({
```

Comment:
+1 on @zarsky-broad's question on api.google.com

I am also seeing this script getting loaded:
https://us.jsagent.tcell.insight.rapid7.com/tcellagent.min.js

from the [tcell file](https://github.com/DataBiosphere/terra-ui/blob/f1f60584e7c2d09e4d49191564df3bdc5343bcb8/src/libs/tcell.js#L7)

It looks like a tool we are or were using, but I am not sure of the context around this one.